cogeo_mosaic/overviews.py

```diff
@@ -42,6 +42,7 @@ def _get_info(asset: str) -> Dict:
         )
 
 
+# TODO: Won't work with DynamoDB
 def _get_asset_example(mosaic_def: Dict) -> str:
     t = list(mosaic_def["tiles"].keys())[0]
     asset = mosaic_def["tiles"][t][0]
```

Comment:
We need to know the `quakdeys` here, and thus it's not possible with DynamoDB. This could be solved by a refactor of the overview function